chore: v1.0.0-beta.87 (#586)
Redocly/openapi-cli: docs/changelog.md

# OpenAPI CLI changelog

## 1.0.0-beta.87 (2022-03-10)

### Fixes

> - Fixed terser crash in client-site bundles due to `process.env` assignment.

Reviewer: ```suggestion
- Fixed an issue with `process.env` assignment that caused crashes in client-side builds.
```
